ledger_sync/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+FROM ubuntu:16.04
+
+RUN echo "deb apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 8AA7AF1F1091A5FD" && \
+    echo 'deb http://repo.sawtooth.me/ubuntu/0.8/stable xenial universe' >> /etc/apt/sources.list && \
+    apt-get update && \
+    apt-get install -y --allow-unauthenticated -q python3-grpcio-tools=1.1.3-1 \
+        python3-pip \
+        python3-sawtooth-sdk
+
+RUN pip3 install rethinkdb
+
+WORKDIR /project/sawtooth-marketplace
+
+# Note that the context must be set to the project's root directory
```

Comment:
Not clear what the project is; is ledger_synch the project or sawtooth-marketplace?

Reply:
The whole marketplace project. It is likely different components will share utility classes (like addressing), and so will all need access to the whole directory.

Reply:
Can we update the comment to say that the context should be set to the sawtooth-marketplace root?